src/main/java/com/hivemq/extensions/aws/cloudwatch/configuration/entities/Config.java

```diff
@@ -49,6 +51,15 @@ public class Config {
     @XmlElement(name = "metric")
     private final @NotNull List<Metric> metrics = new ArrayList<>();
 
+    @XmlElement(name = "zero-values-submission", defaultValue = "false")
+    private final boolean zeroValuesSubmission = false;
+
+    @XmlElement(name = "report-raw-count-value", defaultValue = "false")
+    private final boolean reportRawCountValue = false;
+
+    @XmlElement(name = "aws-endpoint-override")
```

Comment:
Might make sense to also document. Not sure. It is in the hivemq-s3-cluster-discovery-extension.